crates/autopilot/src/database/auction.rs

```diff
@@ -111,4 +135,22 @@ impl Postgres {
         let id = database::auction::replace_auction(&mut ex, &data).await?;
         Ok(id)
     }
+
+    async fn get_quote_interactions(
+        ex: &mut PgConnection,
+        quote_id: QuoteId,
+    ) -> Result<Vec<InteractionData>> {
+        database::quotes::get_quote_interactions(ex, quote_id)
+            .await?
+            .iter()
+            .map(|data| {
+                Ok(InteractionData {
+                    target: H160(data.target.0),
+                    value: big_decimal_to_u256(&data.value)
+                        .context("quote interaction value is not a valid U256")?,
+                    call_data: data.call_data.clone(),
+                })
```

Comment:
Probably make sense to implement `into_domain` function.

Reply:
Yes, I was thinking about that, but I am not sure where `into_domain` should be placed. Adding it to `QuoteInteraction` in `database` (`data` variable type from above code) will introduce dependency on `model` in `database`, currently there is no such dependency.

Reply:
Ah, I see. It belongs to the `model` crate.